### PR TITLE
pythonPackages.bayespy: init at 0.5.10

### PR DESCRIPTION
**Pull request by jluttine**
###### Motivation for this change

Add BayesPy Python package.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/bayespy/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    inherit pname version;
+    sha256 = "01cwd88ri29zy6qpvnqzljkgc44n7a17yijizr73blcnh4dz5n1w";
+  };
+
+  buildInputs = [ nose ];
+  propagatedBuildInputs = [ numpy scipy matplotlib h5py ];
+
+  # We have to move to some other directory, otherwise the tests are run on the
+  # source package, not the installed package. Also, ignore the image comparison
+  # tests as, for some reason, some of them didn't work.
```
Comment (Mic92) [inline pkgs/development/python-modules/bayespy/default.nix]: Is there something wrong with running the tests on the source package?

**Comment by Mic92** [comment]
Is there something wrong with running the tests on the source package?

**Comment by jluttine** [comment]
It just didn't work for some reason. I got a few errors that I had no idea why.

**Comment by jluttine** [comment]
Perhaps it mixes modules from the source package and the installed package as there are both absolute and relative imports in the modules. And that causes some issues.

**Pull request merged**